client/app/dbaas/logs/detail/home/logs-home.html

```diff
@@ -11,48 +11,111 @@
+                            <button
+                                type="button"
+                                class="oui-tile__button oui-button oui-button_link oui-button_full-width"
+                                ng-click="ctrl.goToAllDashboards()">
+                                <span data-ng-bind="::'logs_home_shortcuts_all_dashboard'|translate"></span>
+                            </button>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <oui-tile-definition>
+                        <oui-tile-description>
+                            <a data-ng-href="{{ ctrl.accountDetails.data.graylogWebuiUrl }}"
+                                class="oui-tile__button oui-button oui-link_icon oui-button_link oui-button_full-width"
+                                target="_blank">
+                                <span data-ng-bind="::'logs_home_shortcuts_graylog'|translate"></span>
+                                <i class="oui-icon oui-icon-external_link" aria-hidden="true"></i>
+                            </a>
+                            <a data-ng-href="{{ ctrl.accountDetails.data.graylogApiUrl }}"
+                                class="oui-tile__button oui-button oui-link_icon oui-button_link oui-button_full-width"
+                                target="_blank">
+                                <span data-ng-bind="::'logs_home_shortcuts_graylog_api'|translate"></span>
+                                <i class="oui-icon oui-icon-external_link" aria-hidden="true"></i>
+                            </a>
+                            <div data-ng-if="ctrl.account.data">
+                                <a data-ng-href="{{ ctrl.accountDetails.data.elasticSearchApiUrl }}"
+                                    class="oui-tile__button oui-button oui-link_icon oui-button_link"
+                                    data-ng-class="{
+                                        'oui-link_disabled': !(ctrl.account.data.total.maxNbIndex > 0 || ctrl.account.data.total.maxNbAlias > 0),
+                                        'oui-button_full-width': (ctrl.account.data.total.maxNbIndex > 0 || ctrl.account.data.total.maxNbAlias > 0)
+                                    }"
+                                    target="_blank">
+                                    <span data-ng-bind="::'logs_home_shortcuts_elasticsearch'|translate"></span>
+                                    <i class="oui-icon oui-icon-external_link" aria-hidden="true"></i>
+                                </a>
+                                <oui-popover data-ng-if="!(ctrl.account.data.total.maxNbIndex > 0 || ctrl.account.data.total.maxNbAlias > 0)">
+                                    <button type="button" class="oui-popover__help-button" oui-popover-trigger></button>
+                                    <oui-popover-content>{{::'logs_home_shortcuts_elasticsearch_help'|translate}}</oui-popover-content>
```

Comment:
Is it possible to use `data-translate` instead ?

Reply:
The contents inside <oui-popover-content> tag gets transcluded. Hence its not possible to use data-translate here.